tests/cases/conformance/classes/constructorDeclarations/constructorParameters/readonlyInConstructorParameters.ts

```diff
@@ -0,0 +1,13 @@
+class C {
+    constructor(readonly x: number) {}

```

Comment:
also to clarify if it is just `readonly` without accessibility modifier, are we assuming it is `public`?